.pre-commit-config.yaml

```diff
@@ -22,8 +22,8 @@ repos:
           - "--remove-unused-variables"
 
   # Sort imports
-  - repo: https://github.com/pycqa/isort
-    rev: "5.7.0"
+  - repo: https://github.com/m-kuhn/isort
+    rev: "patch-1"
```

Comment:
@suricactus that's the important change
Upstream PR here: https://github.com/PyCQA/isort/pull/1875

Reply:
Thanks a lot!